src/data/mainnet/celo-tokens-info.json

```diff
@@ -198,6 +198,8 @@
     "decimals": 18,
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/CELO.png",
     "isCoreToken": true,
```

Comment:
What do you think of removing `isCoreToken` from here and setting it for backward compatibility in the cloud function and RTDB write?
This way we don't have to think about it when looking/editing this file.

Reply:
great point, thanks! 
removed `isCoreToken` in favor of programmatically setting, added some tests